[2025-03-10T23:36:42.728+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: project2.insert_employees manual__2025-03-10T23:36:40.618299+00:00 [queued]>
[2025-03-10T23:36:42.743+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: project2.insert_employees manual__2025-03-10T23:36:40.618299+00:00 [queued]>
[2025-03-10T23:36:42.744+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-03-10T23:36:42.744+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-03-10T23:36:42.745+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-03-10T23:36:42.763+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): insert_employees> on 2025-03-10 23:36:40.618299+00:00
[2025-03-10T23:36:42.769+0000] {standard_task_runner.py:55} INFO - Started process 1332 to run task
[2025-03-10T23:36:42.773+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'project2', 'insert_employees', 'manual__2025-03-10T23:36:40.618299+00:00', '--job-id', '183', '--raw', '--subdir', 'DAGS_FOLDER/project2.py', '--cfg-path', '/tmp/tmpw2_4785s']
[2025-03-10T23:36:42.774+0000] {standard_task_runner.py:83} INFO - Job 183: Subtask insert_employees
[2025-03-10T23:36:42.867+0000] {task_command.py:388} INFO - Running <TaskInstance: project2.insert_employees manual__2025-03-10T23:36:40.618299+00:00 [running]> on host 9670e20211e7
[2025-03-10T23:36:42.978+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project2
AIRFLOW_CTX_TASK_ID=insert_employees
AIRFLOW_CTX_EXECUTION_DATE=2025-03-10T23:36:40.618299+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-03-10T23:36:40.618299+00:00
[2025-03-10T23:36:42.979+0000] {sql.py:254} INFO - Executing: CALL update_employees();
[2025-03-10T23:36:42.991+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-10T23:36:43.282+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-10T23:36:43.319+0000] {sql.py:375} INFO - Running statement: CALL update_employees();, parameters: None
[2025-03-10T23:36:43.466+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=project2, task_id=insert_employees, execution_date=20250310T233640, start_date=20250310T233642, end_date=20250310T233643
[2025-03-10T23:36:43.507+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-03-10T23:36:43.537+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
